src/lang/en.json

```diff
@@ -102,6 +103,100 @@
   "filter_win": "Result",
   "filter_lane_role": "Lane",
   "filter_patch": "Patch",
+
```

Comment:
Think they should be removed, otherwise we'll have two copies of each string.  Can you verify that the other language files don't have duplicates as well?